TestCases/optimization_euler/equivalentarea_naca64206/NACA64206.cfg

```diff
@@ -0,0 +1,180 @@
ing | WING, 7, 3, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 3, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 3, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 3, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 0, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 1, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 2, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 3, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 4, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 5, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 6, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 7, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 4, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 0, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 1, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 2, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 3, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 4, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 5, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 6, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 7, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 5, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 0, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 1, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 2, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 3, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 4, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 5, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 6, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 7, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 6, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 0, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 1, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 2, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 3, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 4, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 5, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 6, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 7, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 7, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 0, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 1, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 2, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 3, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 4, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 5, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 6, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 7, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 8, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 9, 8, 1, 0.0, 0.0, 1.0 ); ( 11, 1.0 | Wing | WING, 10, 8, 1, 0.0, 0.0, 1.0 )
```

Comment:
Wait, it is just a single '\'

Reply:
I'll fix this. `gedit` on Ubuntu had difficulty opening this super long line, so it will make it easy to handle this.
This is a copy-and-paste from `set_ffd_design_var.py`, so it might be better to edit this python script so it provides outputs that are easier to handle.